release/sdk-config.yaml.tmpl

```diff
@@ -28,7 +28,7 @@ commands:
 - name: test
   path: damlc/damlc
   args: ["test"]
-  desc: "Run the scenarios in the given Daml file and all dependencies"
```

Comment:
I think the replacing scenarios by scripts is not great: 
- it actually "run" both scenarios + scripts
- it does not really "run" script, to run a script you have to use the command `daml script` rather

I cannot find a good wording, maybe "Run the tests in ..."

Reply:
AFAIU it does run all the scripts it can find. That is, top-level bindings of type `Script a`.

Reply:
I think what @remyhaemmerle-da is getting at is that it runs them against the script service rather than against a "real" ledger. But I’d still stick with calling this "script" and not "test". that’s the terminology we use everywhere else. If you want to make it precise, state that it runs in a "virtual testing ledger" or something like that.

Reply:
Assuming it can be stated succinctly, this may be a good place to specify whether each script runs against its own temp ledger or they all run against the same one, and in the latter case if there's any way for users to influence order.

Reply:
As scripts are a replacement for scenarios in most cases, and scenarios are deprecated, I would avoid mentioning scenario here altogether. So this message looks fine, and it's better than what was there before. I wouldn't change it any more.

Reply:
As scripts are a replacement for scenarios in most cases, and scenarios are deprecated, I would avoid mentioning scenario here altogether. So this message looks fine, and it's better than what was there before. I wouldn't change it any more.